akka-actor-tests/src/test/scala/akka/serialization/SerializeSpec.scala

```diff
@@ -429,13 +429,13 @@ class DefaultSerializationWarningSpec extends AkkaSpec(
   "Using the default Java serializer" must {
 
     "log a warning when serializing classes outside of java.lang package" in {
-      EventFilter.warning(message = messagePrefix) intercept {
+      EventFilter.warning(pattern = messagePrefix, occurrences = 1) intercept {
```

Comment:
Bugfix - "message" is exact match, whereas "pattern" is pattern match. Matching with message**Prefix** should be pattern match.